crates/bevy_ecs/src/archetype.rs

```diff
@@ -99,7 +115,14 @@ pub struct Edges {
 
 impl Edges {
     #[inline]
-    pub fn get_add_bundle(&self, bundle_id: BundleId) -> Option<&AddBundle> {
+    pub fn get_add_bundle(&self, bundle_id: BundleId) -> Option<ArchetypeId> {
```

Comment:
Docs would be nice here: the name is not super helpful at a glance.

Reply:
Documented all of the Edges APIs.